### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by iojw**
Just thought I would add a README since I'm quite familiar with this module 🙂. Do take a look and see if it's alright.

### CODE REVIEW

**Comment by Cervator** [comment]
Looks cool! Will leave this for @xrtariq2594 though since he's the expert :-)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Alteration Effects module
+The Alteration Effects module contains a set of status effects (buffs and debuffs) that can alter a player's base state in some fashion.
+
+These are the `AlterationEffects` that current exist in the module:  
+- `HealthBoostAlterationEffect`- this increases the maximum health of the player by a specified amount
```
Comment (rzats) [inline README.md]: The "this" is redundant here (and in the entire list) IMO. "`HealthBoostAlterationEffect` - increases the maximum health of the player by a specified amount" sounds nicer 🙂

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- `SwimSpeedAlterationEffect` - this increases the speed with which the players swims by a specified amount
+- `WalkSpeedAlterationEffect` - this increases the speed with which the player walks by a specified amount
+
+# Usage
+All `AlterationEffect` objects have two methods that you can use to apply the effects
+
+`applyEffect(EntityRef instigator, EntityRef entity, float magnitude, long duration)`  
+`applyEffect(EntityRef instigator, EntityRef entity, String id, float magnitude, long duration)`
+
+`instigator` - the entity who applies the effect
```
Comment (rzats) [inline README.md]: who → which/that

File: README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+`entity` - the entity the effect is applied on  
+`magnitude` - the magnitude of the effect  
+`duration` -  the duration of the effect, in milliseconds  
+`id` - an optional ID for certain effects which require them (e.g. ResistDamage)
+
+Not all alteration effects will make use of all these parameters. Take a look at the individual classes to see which parameters are used and how.
+
+# Example
+The following example shows how you can initialise a StunAlterationEffect object and apply it on an entity.
+```
```
Comment (rzats) [inline README.md]: Replace this with ` ```java ` for fancy syntax highlighting :D

**Review by rzats — CHANGES_REQUESTED** [review]

**Comment by iojw** [comment]
@rzats: cool, edited based on your comments! thanks for the review (and learnt something new with that fancy syntax highlighting 🙂).

**Review by rzats — APPROVED** [review]

**Pull request merged**